<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>7</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="Go_cliente">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>250</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CLIENTE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Go_admin">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ADMINISTRADOR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>501</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>MENÚ PRINCIPAL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCerrarprogram">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>460</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>SALIR</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="btnCrearReserva">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>160</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CREAR RESERVA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancelarReserva">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR RESERVA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnModificarReserva">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>270</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>MODIFICAR RESERVA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>570</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLineEdit" name="txtNombreCliente">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNumeroContacto">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dtpFechaHoraReserva">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>210</y>
        <width>194</width>
        <height>29</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>15</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>8</month>
        <day>17</day>
       </datetime>
      </property>
     </widget>
     <widget class="QSpinBox" name="spnNumeroComensales">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>270</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConfirmarAgregarReserva">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>AGREGAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancelarAgregarReserva">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>520</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese su nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese su número de contacto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>190</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Defina la fecha y hora:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>250</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el número de comensales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>RESERVACIÓN </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConsultarDisponibilidad">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CONSULTAR DISPONIBILIDAD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>600</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>MODIFICAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese su número de contacto:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spnNumeroComensales_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>290</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNombreCliente_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConfirmarModificarReserva">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>470</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>AGREGAR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNumeroContacto_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese su nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>210</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Defina la fecha y hora:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancelarModificarReserva">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>560</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dtpFechaHoraReserva_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>230</y>
        <width>194</width>
        <height>29</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>15</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>8</month>
        <day>17</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el número de comensales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el ID de referencia:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtReferenciaReserva_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancelar Reservación</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>200</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>INGRESE EL NÚMERO DE REFERENCIA:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtReferenciaReserva_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>290</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancelarReservacion">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>390</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>550</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="btnRESER">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>280</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>RESERVACIONES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTABLES">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>360</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>MESAS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>531</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MENÚ ADMINISTRADOR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Volver_menuprinci">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>440</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>VOLVER</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese su Numero de referencia</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contactLineEdit3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>110</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Fecha A Reservar:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAvailabilityButton3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>550</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>DISPONIBILIDAD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifyReservationButton3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>450</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>MODIFICAR RESERVACIÓN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createReservationButton3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>450</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CREAR RESERVACIÓN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="referenceLineEdit3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit3">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>180</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>10</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>8</month>
        <day>17</day>
       </datetime>
      </property>
     </widget>
     <widget class="QSpinBox" name="guestsSpinBox3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameLineEdit3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero de Contacto:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelReservationButton3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>550</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR RESERVACIÓN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Cantidad de Comensales:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>600</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ID DE MESA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CAPACIDAD DE MESA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddTable_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>180</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Añadir Mesa</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="NumberTablespinBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveTable_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>290</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Eliminar Mesa</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="CapacityspinBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAvailabilityButton4_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>410</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ver Mesas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>570</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
